<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoBaseToolForm</class>
 <widget class="QWidget" name="QWoBaseToolForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>167</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QToolButton" name="btnReturn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="woterm.qrc">
       <normaloff>:/woterm/resource/skin/nail.png</normaloff>:/woterm/resource/skin/nail.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnFullScreen">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="woterm.qrc">
       <normaloff>:/woterm/resource/skin/fullscreen.png</normaloff>:/woterm/resource/skin/fullscreen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnExitFull">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="woterm.qrc">
       <normaloff>:/woterm/resource/skin/exitfull.png</normaloff>:/woterm/resource/skin/exitfull.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnMinimize">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="woterm.qrc">
       <normaloff>:/woterm/resource/skin/minize2.png</normaloff>:/woterm/resource/skin/minize2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnClose">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="woterm.qrc">
       <normaloff>:/woterm/resource/skin/stop2.png</normaloff>:/woterm/resource/skin/stop2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="woterm.qrc"/>
 </resources>
 <connections/>
</ui>
